packages/languages/src/browser/language-client-services.ts

```diff
@@ -27,9 +27,14 @@ export interface Language {
     readonly filenames: Set<string>;
 }
 
+export interface WorkspaceSymbolProvider {
+    provideWorkspaceSymbols(params: services.WorkspaceSymbolParams, token: services.CancellationToken): Thenable<services.SymbolInformation[]>;
```

Comment:
How about `extends services.WorkspaceSymbolProvider` instead of copying a method?